SocialFilter/sections/specific.txt

```diff
@@ -3496,7 +3496,7 @@ babyclub.de##.social-links-container
 censor.net##.footer_theme
 diy.obi.ru##.rating-section-share
 edaily.co.kr##.reporter_sns
-autos.goo.ne.jp,ch-review.net,kuruma-news.jp,alba.co.jp,mskagency.ru,pr.goo.ne.jp,point.md,bike-news.jp,haber7.com,w.atwiki.jp,badische-zeitung.de##.social-button
+basketballking.jp,ruma-news.jp,alba.co.jp,mskagency.ru,pr.goo.ne.jp,point.md,bike-news.jp,haber7.com,w.atwiki.jp,badische-zeitung.de##.social-button
```

Comment:
Mistakenly removed?

Reply:
sorry , i mistook that.

Reply:
Looks `autos.goo.ne.jp,` is still not recovered.

Reply:
sorry.